app/assets/stylesheets/navigation.scss

```diff
@@ -30,32 +23,47 @@ a.button-collapse.top-nav {
 }
 
 .side-nav.fixed {
-  @media #{$large-and-up} {
-    z-index: 1;
-    top: 65px;
-  }
-
   @extend .z-depth-1;
 
   width: $side-nav-width;
 
-  $bg-color: $bottom-color;
-  $font-color: $secondary-color;
+  $bg-color: $secondary-color;
+  $font-color: #FFF;
   $line-height: 64px;
 
+  background: image_url('side-nav.png');
```

Comment:
Name of function `image_url` should be written in all lowercase letters with hyphens instead of underscores<br>Prefer double-quoted strings